webhook/webhook.bal

```diff
@@ -0,0 +1,78 @@
+import ballerinax/trigger.asgardeo;
+import ballerina/log;
+import ballerina/http;
+import ballerinax/scim;
+
+configurable asgardeo:ListenerConfig config = ?;
+configurable string orgName = ?;
+configurable string clientId = ?;
+configurable string clientSecret = ?;
+configurable string contestantGroupId = ?;
+
+scim:ConnectorConfig configscim = {
+    orgName: orgName,
+    clientId: clientId,
+    clientSecret : clientSecret,
+    scope : ["internal_group_mgt_update"]
+};
+
+listener http:Listener httpListener = new(8090);
+listener asgardeo:Listener webhookListener =  new(config,httpListener);
+
+service asgardeo:RegistrationService on webhookListener {
+
+    remote function onAddUser(asgardeo:AddUserEvent event ) returns error? {
+        json user = event.toJson();
+        json|error userName = user.eventData.userName;
+        json|error userId = user.eventData.userId;
```

Comment:
without converting to JSON, shall we get the `username` and `userId` from the event record? like,
`event. userName`
`event. userId`